airflow/www/templates/airflow/graph.html

```diff
@@ -169,10 +169,26 @@
                                         "scale(" + d3.event.scale + ")");
           });
       svg.call(zoom);
-    }
 
-    // Centering the DAG on load (Not implemented yet)
-    // https://github.com/dagrejs/dagre-d3/issues/245
+      // Centering the DAG on load
+      // Get Dagre Graph dimensions
+      var graphWidth = g.graph().width;
+      var graphHeight = g.graph().height;
+      // Get SVG dimensions
+      var padding = 20;
+      var width = parseInt(svg.style("width").replace(/px/, "")) - padding*2;
```

Comment:
Does this work if the width is defined in units other than px in the css?

Reply:
i believe s3 always sets the style in px. but i found a better method to get the real dimension of the svg element, so i have switched to that, which returns  px as integer.